script/iframe-messaging.js

```diff
@@ -13,4 +13,8 @@ ANSWERS.core.storage.registerListener({
       sessionId: ANSWERS.core.getOrSetupSessionId()
     }));
   }
-});
\ No newline at end of file
+});
+
+window.parentIFrame.sendMessage(JSON.stringify({
```

Comment:
iI looks like this sends a message from the iframe to the parent, and then back to the iframe. Is this so that we know that Answers is initialized by the time we update the URLs? If so, I'm wondering if it makes sense to change this message to something like:
```
{ 
    body: 'answers-initialized'
}
```
That way the iframe-common code can send the update-tab-navigation back to the iframe once it gets the answers-initialized message.

Reply:
yes this is to indicate that answers has been initialized so we can update the URL in iframeresizer. Sure, I can update the message action!